src/components/templates/Playa/PlayaVenue.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useRef } from "react";
 import { Overlay } from "react-bootstrap";
 import { DEFAULT_MAP_ICON_URL, PLAYA_VENUE_SIZE } from "settings";
-import { AnyVenue } from "types/Firestore";
+import { AnyVenue } from "types/venues";
```

Comment:
I thought Playa is completely removed from our codebase?

Reply:
See https://github.com/sparkletown/sparkle/pull/1080#discussion_r564114898